examples/flow_box/main.rs

```diff
@@ -0,0 +1,61 @@
+mod data_set;
+use std::str::FromStr;
+
+use gtk::gdk;
+use gtk::prelude::*;
+
+fn create_color_button(color: &'static str) -> gtk::Button {
+    let button = gtk::Button::new();
+    let drawing_area = gtk::DrawingAreaBuilder::new()
+        .content_height(24)
+        .content_width(24)
+        .build();
+    drawing_area.set_draw_func(move |_, cr, _width, _height| {
+        let rgba = gdk::RGBA::from_str(color).unwrap();
```

Comment:
This should probably happen outside the closure, once. Instead of inside the closure for each frame that is drawn :)

Reply:
Oh indeed.